doc/development/plugins.rst

```diff
@@ -61,7 +61,7 @@ and subclassing it:
 .. warning::
 
     The API of PennyLane devices will be updated soon to follow a new interface defined by
-    the experimental :class:`pennylane.devices.experimental.Device` class. This guide describes
+    the :class:`pennylane.devices.Device` class. This guide describes
     how to create a device with the current :class:`pennylane.Device` and
     :class:`pennylane.QubitDevice` base classes, and will be updated as we switch to the new API.
```

Comment:
Probably for a different PR, but this should be updated for 0.33, right?

Reply:
indeed, I opened a story for it when I saw this in the `0.33 Stories` epic

Reply:
updated the story to also mention adding_operators.rst